decidim-core/spec/lib/features/current_feature_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "spec_helper"
 
 module Decidim
-  describe CurrentComponent do
```

Comment:
Looks like your previous PR removed the specs for the `CurrentOrganization` middleware (see [history](https://github.com/AjuntamentdeBarcelona/decidim/blob/6fa0a54a63de6dd2ed97b16b70e8cd7ed56fb5a7/decidim-core/spec/middleware/decidim/current_organization_spec.rb)). 

Can you readd them please?